common/app/common/configuration.scala

```diff
@@ -474,6 +474,21 @@ class GuardianConfiguration extends Logging {
     lazy val config = configuration.getMandatoryStringProperty("front.config")
   }
 
+  object targeting {
+    lazy val campaignsUrl = configuration.getStringProperty("targeting.campaignsUrl")
+
+    // Total number of rules allowed per campaign, any campaigns with more than one rule will be filtered
+    lazy val ruleLimit: Int =
+      Try(configuration.getStringProperty("targeting.ruleLimit").get.toInt)
```

Comment:
we already have a lot of config, in my opinion we should save it for private stuff and things that must differ between code and prod where possible, otherwise we end up writing the code in configuration to avoid compiling